src/main/java/seedu/mtracker/console/AddInstrumentParser.java

```diff
@@ -64,9 +64,9 @@ public static String getInstrumentSentimentFromUser() {
 
 
     public static void addSentimentToParameters() {
-        String sentiment = getInstrumentSentimentFromUser();
+        String sentiment = getInstrumentSentimentFromUser().toLowerCase();
         while (!Validate.isValidSentiment(sentiment)) {
-            sentiment = getInstrumentSentimentFromUser();
+            sentiment = getInstrumentSentimentFromUser().toLowerCase();
         }

```

Comment:
Change to do-while loop.

Reply:
i think can merge theodore's pr on do while loop then ill merge this after or the other way arnd

Reply:
Ok yours can go first then.